presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -39,6 +41,7 @@
         private final SettableLookupSourceSupplier lookupSourceSupplier;
         private final List<Integer> hashChannels;
         private final Optional<Integer> hashChannel;
+        private final Optional<JoinFilterFunction> filterFunction;
```

Comment:
I would squash this commit with one before. As there you introduced an entity which is used here. When squashed, reader have a full story of this entity (JFF).